src/stcal/ramp_fitting/ramp_fit_class.py

```diff
@@ -0,0 +1,118 @@
+class RampFitInternal:
```

Comment:
I think this would work, but then we would have JWST ramp models, Roman ramp models, and these internal general ramp models. All of these are quite similar, so perhaps they could instead be inherited from a parent ramp fit class, importable from within jwst, romancal, and stcal.  And within this new class structure we could incorporate the mission-specific DQ flags information, instead of having a new parameter.

Reply:
The reason I created this simple class is because data models are too complex to be pickled.  The point of this PR is to create a simple class containing only the information needed for ramp fitting computation in order to be pickled in preparation for using the `multiprocessing` module.

Reply:
Yeah, I know that data models can't be pickled.

Now that I've looked at this more closely, I understand how this class can be used for multiprocessing.

Reply:
I changed the class name to `RampData`.